remix-analyzer/src/solidity-analyzer/index.ts

```diff
@@ -1,25 +1,31 @@
 'use strict'
 import { AstWalker } from 'remix-astwalker'
 import list from './modules/list'
+import { CompilationResult, AnalyzerModule, AnalysisReportObj, AnalysisReport } from 'types'
+
+type ModuleObj = {
+  name: string
+  mod: AnalyzerModule
+}
 
 export default class staticAnalysisRunner {
 
-  run (compilationResult, toRun, callback) {
-    const modules = toRun.map((i) => {
-      const m = this.modules()[i]
+  run (compilationResult: CompilationResult, toRun: any[], callback: ((reports: AnalysisReport[]) => void)): void {
+    const modules: ModuleObj[] = toRun.map((i) => {
+      const m: AnalyzerModule = this.modules()[i]
       return { 'name': m.name, 'mod': m }
     })
-
     this.runWithModuleList(compilationResult, modules, callback)
   }
 
-  runWithModuleList (compilationResult, modules, callback) {
-    let reports: any[] = []
+  runWithModuleList (compilationResult: CompilationResult, modules: ModuleObj[], callback: ((reports: AnalysisReport[]) => void)): void {
+    let reports: AnalysisReport[] = []
```

Comment:
It doesn't seem like you're reassigning the value later (just mutate it). You can use `const`

Reply:
This is rewritten here: https://github.com/ethereum/remix/pull/1427/files/f608f857abb95444134404844dbff8a222f24fab#diff-ae3e78deb6c266cd1f728d3cd3a749ebR46